### ISSUE TITLE
Player names are shortened on the HUD when a kill goal is set or when in co-op

### ISSUE THREAD

**Issue by Mako88** [dxx-rebirth/dxx-rebirth#135]
The max number of characters in a player's name shown seems to be 4 or 5 when either a kill goal is set or when playing in co-op.

**Comment by vLKp** [comment]
As I read the code, this happened in 0.58.1-d2x also.

**Comment by zicodxx** [comment]
I don't think this is a bug but intentionally done so the score/kill strings do not overlap the player name. I guess the main focus was that the whole string (callsign + score) still fits into the the side window of the cockpit view. So more of an aesthetic than a practical decision.

**Comment by Mako88** [comment]
Is there a way to only shorten it on views that would cause it to overlap? So not shorten it on fullscreen, but still shorten it on the cockpit view?

**Comment by vLKp** [comment]
There are no configuration options for it.  A heuristic or a configuration option would be possible, since the goal is mostly to bypass the code that truncates the name.
